mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -408,6 +451,119 @@ private[classification] class MultiClassSummarizer extends Serializable {
 }
 
 /**
+ * Abstraction for multinomial Logistic Regression Training results.
+ */
+sealed trait LogisticRegressionTrainingSummary extends Serializable {
+
+  /** objective function (scaled loss + regularization) at each iteration. */
+  val objectiveHistory: Array[Double]
```

Comment:
I'd prefer this be marked as a def.  Not sure if it matters, but that's what I've seen done.